api/db-cdr/generate-cdr/build-prep-survey.sh

```diff
@@ -18,20 +25,6 @@ QUESTION_PARENT_ID=0
 ANSWER_PARENT_ID=0
 OUTPUT_FILE_NAME=$(echo "$FILE_NAME" | cut -d'_' -f 1 | xargs -I {} bash -c 'echo {}.csv')
 
-function check_prep_survey() {
```

Comment:
you dont think there will be a case where the $CREATE_SURVEYS=false but the prep_survey row count is 0?
May be do a sanity check for select count(*) from [dataset].prep_survey is needed?